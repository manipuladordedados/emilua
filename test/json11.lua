local json = require('json')

local function my_print(x)
    print(type(x), x)
end

my_print(json.decode('1'))
my_print(json.decode('1.5'))
my_print(json.decode('false'))
my_print(json.decode('true'))
my_print(json.decode('null'))
my_print(json.decode('"hello world"'))
