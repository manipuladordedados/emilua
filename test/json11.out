number	1
number	1.5
boolean	false
boolean	true
table	null
string	hello world
